clang/test/CXX/drs/cwg28xx.cpp

```diff
@@ -47,6 +47,31 @@ void f() {
 #endif
 } // namespace cwg2813
 
+namespace cwg2815 { // cwg2815: 21
+#if __cplusplus >= 201703L
+int arg() noexcept;
+
+int f(int (&)() noexcept);
+void f(int (&)());
+int i = f(arg);
+
+int g(int (*)() noexcept);
+void g(int (&)());
+int j = g(arg);
+
+int h(int (&)() noexcept);
+void h(int (*)());
+int k = h(arg);
+
+int a(int (*)());
+int a(int (&)());
+int x = a(arg);
+// expected-error@-1 {{call to 'a' is ambiguous}}
+//   expected-note@-4 {{candidate function}}
```

Comment:
```suggestion
//   expected-note@#cwg2815-a-ptr {{candidate function}}
```
As you've experienced yourself, maintaining relative line offsets is annoying. Counting lines outside of very trivial cases it annoying for reviewers, too.